### ISSUE TITLE
Confusing (?) error message for slide-in

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete-intellij-plugin#451]
This error message is somewhat confusing:

![image](https://user-images.githubusercontent.com/3383210/88594308-410d0980-d061-11ea-8c2c-0d641db49946.png)

1. Maybe just `branch` instead of `entry`
1. Apostrophes used/not used inconsistently
1. To make the things more clear, maybe reword the message to sth like `Branch 'develop' cannot be slid under branch 'bugfix/...' since 'develop' is already an ancestor of 'bugfix/...'`

**Comment by mkondratek** [comment]
Ad 3.
We are repeating here (both branch names twice).
`Cannot slide in since branch 'develop' is already an ancestor of the branch 'bugfix/...` 
WDYT?

**Comment by PawelLipski** [comment]
Use your judgement :) but optimize for clarity rather than style
